projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,37 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: block;
+    width: 1rem;
+    height: 1rem;
+    border-radius: 100%;
+    box-shadow: 0 0 0.4375rem rgba(0, 0, 0, 0.3);
+    border-color: var(--tui-base-02);
+    color: var(--tui-primary);
+    box-sizing: border-box;
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: 0.125rem;
+        left: 0.125rem;
+        background: currentColor;
+        mask-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg"><path d="M0 8.99993C4 8 8 8 12 8.99993C9.91509 5.73239 8.5 3 6 0C3.5 3 2 5.5 0 8.99993Z"/></svg>');
+        width: 0.75rem;
+        height: 0.75rem;
+        transform: rotate(var(--t-degrees)) translateY(-0.5rem);
+    }
+
+    &:after {
+        .fullsize();
```

Comment:
`.fullsize(absolute, inset)` so that it uses `right` and `bottom` instead of `width` and `height` thus rendering `box-sizing` unnecessary.

Reply:
fixed